### PR TITLE
Update last 5 Cadence tutorials

### PR DESCRIPTION
**Pull request by sisyphusSmiling**
Related: #62 

Updating second half of Cadence tutorials to align with Playground v2 layout, fixing any dead links, and updating verbiage.

### CODE REVIEW

**Comment by franklywatson** [comment]
Lots of great updates! Thanks!

**Review by franklywatson — APPROVED** [review]

**Review by joshuahannan — COMMENTED** [review]
The wording and everything looks good, but I'd prefer to leave the callout boxes in for the instructions. Feel free to keep the steps numbered if you want, but lets keep them in callout boxes so that they show up once the docs site is fixed.

Also the vercel deployment is failing, so we also need to make sure that succeeds before we merge

**Comment by sisyphusSmiling** [comment]
Thanks @joshuahannan! Waiting on my Vercel access request to be able to see the details of the failed deployment

File: docs/cadence/tutorial/06-fungible-tokens.mdx [hunk 1]
```diff
@@ -292,7 +292,7 @@ If an address doesn’t have the correct resource type imported, the transaction
 
 **Important note: This protection is not in place for the Flow network currency,**
 **because every Flow account is initialized with a default Flow Token Vault**
-**in order to pay for [storage fees and transaction fees](../../concepts//flow-token/concepts.md#fees).**
+**in order to pay for [storage fees and transaction fees](https://developers.flow.com/flow-token/concepts#fees).**
```
Comment (bthaile) [inline docs/cadence/tutorial/06-fungible-tokens.mdx:295]: Let's use relative links instead of absolute links.

**Review by bthaile — APPROVED** [review]
use relative link comment, 
other than that lgtm

**Comment by nialexsan** [comment]
@sisyphusSmiling could you please check callout tags, some of them don't have matching closing tags or there are extra closing tags without matching opening ones

**Comment by sisyphusSmiling** [comment]
@nialexsan thanks, looks like I managed to clean up any unmatched callout tags. Should be GTG now

**Review by joshuahannan — APPROVED** [review]

**Pull request merged**